OpenEdXMobile/src/main/java/org/edx/mobile/view/AccountFragment.kt

```diff
@@ -54,7 +69,18 @@ class AccountFragment : BaseFragment() {
     @Inject
     lateinit var userService: UserService
 
+    @Inject
+    lateinit var iapDialog: InAppPurchasesDialog
+
     private var getAccountCall: Call<Account>? = null
+    private var verifiedCoursesSku: List<String> = arrayListOf()
```

Comment:
we can remove these values. 
`verifiedCoursesSku` & `incompletePurchases`

Reply:
removed.